src/api/goods.ts

```diff
@@ -3,17 +3,27 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export function getAll():Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(new Error('Can`t load Goods'));
```

Comment:
if you return an error here, you need also process it while calling `getAll()` function in `App` component